Rakefile

```diff
@@ -157,7 +158,11 @@ task 'gem:native', [:plat] do |t, args|
   prepare_ccache_cmd += "source tools/internal_ci/helper_scripts/prepare_ccache_symlinks_rc "
 
   supported_windows_platforms = ['x86-mingw32', 'x64-mingw32', 'x64-mingw-ucrt']
-  supported_unix_platforms = ['x86_64-linux', 'x86-linux', 'aarch64-linux', 'x86_64-darwin', 'arm64-darwin']
+  supported_unix_platforms = [
+    'x86_64-linux-gnu', 'x86_64-linux-musl', 'x86-linux-gnu',
```

Comment:
can we please update `unix_platforms_without_debug_symbols` to include everything besides `x86_64-linux-gnu`?

(we can add other platforms later if/when we can get debug symbols working for them)